canonical-web-and-design/ubuntu.com
# Add redirects to fix sentry errors

## Done

- This fixes 2 sentry errors:
    - links to ubuntu.com/20.10 - https://sentry.is.canonical.com/canonical/ubuntu-com/issues/8009/?query=is%3Aunresolved
    - robots are crawling back from a blog topic page to see if there is anything at /blog/topic and /blog/topics, so this fixes that blog/topic/?: "/blog" - https://sentry.is.canonical.com/canonical/ubuntu-com/issues/10871/?query=is%3Aunresolved

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
   - see that http://0.0.0.0:8001/20.10 goes to /download
   - see that http:/0.0.0.0:8001/blog/topic and /blog/topics now goes to /blog
